tutorials/fundamentals/compare-without-time.md

```diff
@@ -0,0 +1,30 @@
+If you want to compare two dates in JavaScript without using the time aspect, you should use
```

Comment:
Minor note: for the future, let's use the convention of one sentence per line. So everything from "If you want" to "the `toDateString()` method." should be on line 1, then "It returns the date" through "as a string" on line 2, and so on.

This makes editting easier and also incentivizes us to keep sentences short :)